lib/collection.js

```diff
@@ -1075,6 +1092,17 @@ Collection.prototype.updateMany = function(filter, update, options, callback) {
     options.ignoreUndefined = this.s.options.ignoreUndefined;
   }
 
+  var keys = Object.keys(update);
+
+  // same errors as the server would give for update doc lacking atomic operators
+  if (keys.length == 0) {
+    return callback(
+      toError('The update operation document must contain at least one atomic operator.')
+    );
+  } else if (keys[0][0] != '$') {
```

Comment:
nitpick: similarly to the first comment, because you're already returning there's no real reason for the else statement: 

```
if (keys.length === 0) {
  return callback(
    toError('The update operation document must contain at least one atomic operator.')
  );
}

if (keys[0][0] !== '$') {
  return callback(toError('the update operation document must contain atomic operators.'));
}
```